packages/react-core/src/components/Page/Page.tsx

```diff
@@ -254,24 +278,58 @@ export class Page extends React.Component<PageProps, PageState> {
     }
 
     let crumb = null;
+
     if (breadcrumb && isBreadcrumbWidthLimited) {
       crumb = (
-        <section className={css(styles.pageMainBreadcrumb, styles.modifiers.limitWidth)}>
+        <section
+          className={css(
+            styles.pageMainBreadcrumb,
+            styles.modifiers.limitWidth,
+            formatBreakpointMods(
+              breadcrumbProps?.stickyOnBreakpoint,
+              styles,
+              'sticky-',
+              getVerticalBreakpoint(height),
+              true
+            )
+          )}
+        >
           <div className={css(styles.pageMainBody)}>{breadcrumb}</div>
         </section>
       );
     } else if (breadcrumb) {
-      crumb = <section className={css(styles.pageMainBreadcrumb)}>{breadcrumb}</section>;
+      crumb = (
+        <section
+          className={css(
+            styles.pageMainBreadcrumb,
+            formatBreakpointMods(
+              breadcrumbProps?.stickyOnBreakpoint,
+              styles,
+              'sticky-',
+              getVerticalBreakpoint(height),
+              true
+            )
+          )}
+        >
+          {breadcrumb}
+        </section>
+      );
```

Comment:
Not sure if you could combine these two, but the only difference should be that `isBreadcrumbWidthLimited` will add `styles.modifiers.limitWidth` to `pageMainBreadcrumb` and wrap `{breadcrumb}` in `<div className={css(styles.pageMainBody)} />`

Reply:
Ah great suggestion! Thanks @mcoker